unit_tests/test_manage_user.py

```diff
@@ -0,0 +1,49 @@
+from pathlib import Path
+
+from fixtures import TestWithFixtures, TempDir
+
+from importlib.machinery import SourceFileLoader
+
+
+# manually import the script since it's not in a module
+manage_user = SourceFileLoader(
+    'manage_user', "resources/manage-user").load_module()
```

Comment:
I prefer to have all resources in one place, it's easier to track what gets copied.